### PR TITLE
I have fixed your readme for you. :)

### PR DESCRIPTION
**Pull request by Saltywan**
https://www.youtube.com/watch?v=QAZc9xsQNjQ
🗡️ 
🍶

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,19 @@
 <p align="left"> 
-  <img src="https://komarev.com/ghpvc/?username=saltywan&label=Profile%20views&color=0e75b6&style=flat" alt="Views" />&nbsp;               
+  <img src="https://komarev.com/ghpvc/?username=toptheworst&label=Profile%20views&color=0e75b6&style=flat" alt="Views" />&nbsp;               
 </p>
 
 ### Hello,
 <!-- - 👋 I’m urmom -->
-- 👀 I’m interested in Web Development, AI/Machine learning, Cybersecurity 
-- 🌱 I’m currently learning Machine Learning
+- 👀 I’m interested in Pharmacy and Space Station 13
```
Comment (TopTheWorst) [inline README.md:7]: bro forgor the real stuff 💀💀💀💀💀 where ma interested in funni business stuff at?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,19 @@
...
+- 🌱 I’m currently learning Thai
```
Comment (TopTheWorst) [inline README.md]: nah, im learning thaiglish, get this right

File: README.md [hunk 3]
```diff
@@ -1,54 +1,19 @@
...
-
-### Machine Learning / Deep Learning:
-<div>
-  <img src="https://img.shields.io/badge/scikit_learn-F7931E?style=for-the-badge&logo=scikit-learn&logoColor=white" title="sklearn" alt="sklearn"/>&nbsp;
-  <img src="https://img.shields.io/badge/Keras-FF0000?style=for-the-badge&logo=keras&logoColor=white" title="keras" alt="keras"/>&nbsp;
-  <img src="https://img.shields.io/badge/TensorFlow-FF6F00?style=for-the-badge&logo=TensorFlow&logoColor=white" title="tensorflow" alt="tensorflow"/>&nbsp;
-  <img src="https://img.shields.io/badge/PyTorch-EE4C2C?style=for-the-badge&logo=pytorch&logoColor=white" title="pytorch" alt="pytorch"/>&nbsp;
-
-</div>
+## Skill Issue
```
Comment (TopTheWorst) [inline README.md]: you forgot to add "none" in this section

File: README.md [hunk 4]
```diff
@@ -1,54 +1,19 @@
...
 
 ---
 ## Stats:
-![GitHub stats](https://github-readme-stats.vercel.app/api?username=saltywan&show_icons=true&theme=tokyonight)
-<p><img align="left" src="https://github-readme-stats.vercel.app/api/top-langs?username=saltywan&show_icons=true&locale=en&layout=compact&theme=tokyonight" alt="urmom" /></p>
+- **None**
```
Comment (TopTheWorst) [inline README.md:19]: false

**Review by TopTheWorst — CHANGES_REQUESTED** [review]
I cant believe u missed this

**Comment by Saltywan** [comment]
I will prompty fix this. Thank you for addressing this.

**Comment by Saltywan** [comment]
# Please recheck :D

**Comment by TopTheWorst** [comment]
ehhhh good enough i guess whatever

**Comment by TopTheWorst** [comment]
ops wrong buton

**Pull request merged**